frontend/src/components/auth/AuthFormWrapper.jsx

```diff
@@ -68,6 +73,18 @@ function AuthFormWrapper({isModalOpen}) {
   };
 
 
+  const sendCodeToVerifyAccount = async () => {
+    toast.promise(getCodeToVerifyAccount(form.email), {
+      loading: "Sending code",
+      success: (code)=>{
+        setVerificationCode(code)
+        toast.info("The code has been sent,if you don't receive it, check if your email is valid")
```

Comment:
You're doing a validation in the "email" text field to ensure the email is valid, so this error message is a little redundant.

Reply:
That is because if the user has inserted an email that does not exist, they must review it to insert it correctly or make sure that it exists, validation only ensures that they follow the guidelines of how an email should be written.

Reply:
Yes, you are right. It should be updated something like "check if your email is valid" to "it may not exist"